theme/static/style/color-palette-light.css

```diff
@@ -22,6 +22,17 @@ nav.primary a.active, #pagination a.active {
     color: #e3edea;
 }
 
+.mobilenav ul a:hover{
+    background-color: #1c3d32;
+    color: #fff;
+}
+
+@media screen and (max-width: 991px) {
+    .mobilenav {
+        background-color: #9ab451;
+    }
+}
```

Comment:
move this definition to the top, just below `.primary` where we define the colors.